types/iron/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
```

Comment:
Just enable `strict`.

Reply:
Good tip! I enable it, but now Travis is complaining:
[Error: Unexpected compiler option strict](https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/318371571?utm_source=github_status&utm_medium=notification)

Please, any idea?

Reply:
Solved! Thanks!